### ISSUE TITLE
`wrapping`: raw string false positive `Missing newline after "{"`

### ISSUE THREAD

**Issue by bcmedeiros** [pinterest/ktlint#1776]
I have some code using `let` blocks inside raw strings, something like:
```
package test

class QueryGenerator {
    fun getQueryString(query: QueryRequest): String {
        val q = """
            SELECT *
            FROM table
            WHERE 1 = 1
            ${query.gameId?.let { "AND id = ?" } ?: ""}    
        """
        return q
    }
}

data class QueryRequest(
    val gameId: String?,
)
```

## Expected Behavior
This should cause no violations of the `wrapping` rule, regardless of the string size.

## Observed Behavior
In the code above, if I replace `table` by `table_name_long_enough_so_max_line_length_is_met_and_wrapping_rule_is_violated_and_generates_an_error`, for example (anything that makes the raw string size go above `max_line_length` seems to work), then I start seeing:
```
QueryGenerator.kt:9:34: Missing newline after "{" (wrapping)
```

## Steps to Reproduce
1. Start with the code above
2. Make the raw string big enough
3. Run ktlint 0.48+ in the file

## Your Environment

* Version of ktlint used: 0.48.1
* Relevant parts of the `.editorconfig` settings: `max_line_length = 160`
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): ktlint CLI
* Operating System and version: Arch Linux (rolling as of 2023-01-15)

**Comment by bcmedeiros** [comment]
This issue is very tricky to reproduce, actually, this might not be related to `max_line_length` at all. I just tried moving the entire content of the raw string to a single line and the error is gone, even with a very long raw string.
The exact values I provided causes the issue 100% of the times I tried, though, so I hope that's enough to reproduce the issue on your side.

**Comment by paul-dingemans** [comment]
Can be reproduced. The error is introduced in `0.48.x`.

**Comment by bcmedeiros** [comment]
Thanks for the quick reply :)
This is my only blocker to update to 0.48, so let me know if I can help with anything.

**Comment by paul-dingemans** [comment]
You can try to solve this issue via a PR ;-)

Otherwise. just disable the rule (https://pinterest.github.io/ktlint/faq/#how-do-i-suppress-errors-for-a-lineblockfile) and start using the new version.

**Comment by bcmedeiros** [comment]
I can try, but probably just next weekend.
Iḿ not in a rush to upgrade, just wanted to make sure the issue was reported as soon as possible.

**Comment by paul-dingemans** [comment]
@bcmedeiros Now I recognize your user name from the Spotless project. Does this issue block Spotless or a personal project? If it blocks Spotless, then I will have a look my self because I want to release a bugfix release of ktlint within a few days.

**Comment by bcmedeiros** [comment]
> Does this issue block Spotless or a personal project? 

No, it blocks the project Iḿ currently in charge of.

I see you fixed it, anyway, thank you very much!

Looking forward to the release, let me know if you want me to do some testing.

**Comment by bcmedeiros** [comment]
Was this included in 0.48.2? I think I missed it in the release notes.

**Comment by paul-dingemans** [comment]
It should be included. It looks like that something gone wrong with updating the changelog in the commit: https://github.com/pinterest/ktlint/pull/1779/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR14

**Comment by bcmedeiros** [comment]
Yeah, I can confirm the error is gone, thanks!
